R/ospsuite-env.R

```diff
@@ -1,3 +1,11 @@
+
+.getSuiteVersion <- function() {
+  version <- getNamespaceVersion("ospsuite")

```

Comment:
@Yuri05 @PavelBal This is in fact the better way apparently to load the version. Otheriwse, if you have two version installed, the proposed packageVersion reads the first one all the time. This reads the one loaded

Reply:
Looks good!